internal/renderers/femtovg/itemrenderer.rs

```diff
@@ -211,13 +211,42 @@ fn draw_glyphs<R: femtovg::Renderer + TextureImporter>(
 
                     let brush = glyph_run.style().brush;
 
-                    let glyphs = glyph_run.positioned_glyphs().map(|glyph: parley::Glyph| {
-                        femtovg::PositionedGlyph {
+                    let run_end = glyph_run.offset() + glyph_run.advance();
+
+                    let needs_elide = layout
+                        .elision_info
+                        .as_ref()
+                        .filter(|info| run_end > info.max_physical_width);
+
+                    let filtered_glyphs: Vec<_> = glyph_run
+                        .positioned_glyphs()
+                        .filter(|glyph| {
+                            if let Some(info) = needs_elide {
+                                if glyph.x + glyph.advance + info.laid_out_elipsis.advance
+                                    > info.max_physical_width
+                                {
+                                    return false;
+                                }
+                            }
+
+                            true
+                        })
+                        .collect();
```

Comment:
Drive-by thought: This logic could probably be shared in `sharedparley` as we'll need it in all renderers.

Reply:
Yep! Done that now. Will merge when CI is green-ish